sweep/walletsweep.go

```diff
@@ -208,27 +210,28 @@ type DeliveryAddr struct {
 }
 
 // CraftSweepAllTx attempts to craft a WalletSweepPackage which will allow the
-// caller to sweep ALL outputs within the wallet to a list of outputs. Any
-// leftover amount after these outputs and transaction fee, is sent to a single
-// output, as specified by the change address. The sweep transaction will be
-// crafted with the target fee rate, and will use the utxoSource and
-// outputLeaser as sources for wallet funds.
+// caller to sweep ALL funds in ALL or SELECT outputs within the wallet to a
+// list of outputs. Any leftover amount after these outputs and transaction fee,
+// is sent to a single output, as specified by the change address. The sweep
+// transaction will be crafted with the target fee rate, and will use the
+// utxoSource and outputLeaser as sources for wallet funds.
 func CraftSweepAllTx(feeRate, maxFeeRate chainfee.SatPerKWeight,
 	blockHeight uint32, deliveryAddrs []DeliveryAddr,
 	changeAddr btcutil.Address, coinSelectLocker CoinSelectionLocker,
 	utxoSource UtxoSource, outputLeaser OutputLeaser,
-	signer input.Signer, minConfs int32) (*WalletSweepPackage, error) {
+	signer input.Signer, minConfs int32,
+	selectUtxos ...wire.OutPoint) (*WalletSweepPackage, error) {
```

Comment:
I think the diff is fairly simple,
```diff
diff --git a/rpcserver.go b/rpcserver.go
index 80cff0763..619c635a8 100644
--- a/rpcserver.go
+++ b/rpcserver.go
@@ -1389,7 +1389,7 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 		sweepTxPkg, err := sweep.CraftSweepAllTx(
 			feePerKw, maxFeeRate, uint32(bestHeight), nil,
 			targetAddr, wallet, wallet, wallet.WalletController,
-			r.server.cc.Signer, minConfs, selectOutpoints...,
+			r.server.cc.Signer, minConfs, selectOutpoints,
 		)
 		if err != nil {
 			return nil, err
@@ -1444,7 +1444,7 @@ func (r *rpcServer) SendCoins(ctx context.Context,
 				outputs, targetAddr, wallet, wallet,
 				wallet.WalletController,
 				r.server.cc.Signer, minConfs,
-				selectOutpoints...,
+				selectOutpoints,
 			)
 			if err != nil {
 				return nil, err
diff --git a/sweep/walletsweep.go b/sweep/walletsweep.go
index f0ec70e7c..1fa2068ea 100644
--- a/sweep/walletsweep.go
+++ b/sweep/walletsweep.go
@@ -220,7 +220,7 @@ func CraftSweepAllTx(feeRate, maxFeeRate chainfee.SatPerKWeight,
 	changeAddr btcutil.Address, coinSelectLocker CoinSelectionLocker,
 	utxoSource UtxoSource, outputLeaser OutputLeaser,
 	signer input.Signer, minConfs int32,
-	selectUtxos ...wire.OutPoint) (*WalletSweepPackage, error) {
+	selectUtxos []wire.OutPoint) (*WalletSweepPackage, error) {
 
 	// TODO(roasbeef): turn off ATPL as well when available?
 
diff --git a/sweep/walletsweep_test.go b/sweep/walletsweep_test.go
index 8cc93c941..724b9d85d 100644
--- a/sweep/walletsweep_test.go
+++ b/sweep/walletsweep_test.go
@@ -346,7 +346,7 @@ func TestCraftSweepAllTxCoinSelectFail(t *testing.T) {
 
 	_, err := CraftSweepAllTx(
 		0, 0, 10, nil, nil, coinSelectLocker, utxoSource, utxoLeaser,
-		nil, 0,
+		nil, 0, nil,
 	)
 
 	// Since we instructed the coin select locker to fail above, we should
@@ -372,7 +372,7 @@ func TestCraftSweepAllTxUnknownWitnessType(t *testing.T) {
 
 	_, err := CraftSweepAllTx(
 		0, 0, 10, nil, nil, coinSelectLocker, utxoSource, utxoLeaser,
-		nil, 0,
+		nil, 0, nil,
 	)
 
 	// Since passed in a p2wsh output, which is unknown, we should fail to
@@ -461,7 +461,7 @@ func TestCraftSweepAllTx(t *testing.T) {
 			sweepPkg, err := CraftSweepAllTx(
 				0, 0, 10, nil, deliveryAddr, coinSelectLocker,
 				utxoSource, utxoLeaser, signer, 0,
-				tc.selectUtxos...,
+				tc.selectUtxos,
 			)
 			if tc.errString != "" {
 				require.ErrorContains(t, err, tc.errString)
```